YotiButtonSDK/Kernel/KernelSDK.swift

```diff
@@ -20,11 +20,11 @@ class KernelSDK: NSObject {
     /**
      * Perform a call to the Yoti API to retrieve the scenario and start the Yoti App
      */
-    func startScenario(for useCaseID: String) {
+    public func startScenario(for useCaseID: String, with delegate: YotiSDKDelegate) {
```

Comment:
I don't think you should make this public as the `KernelSDK` is an internal class.
(Same for the functions below)